src/expr/src/scalar/func.rs

```diff
@@ -1626,6 +1627,19 @@ fn map_get_value<'a>(a: Datum<'a>, b: Datum<'a>) -> Datum<'a> {
     }
 }
 
+fn list_contains_list<'a>(a: Datum<'a>, b: Datum<'a>) -> Datum<'a> {
+    let mut counts = BTreeMap::new();
+
+    a.unwrap_list()
+        .iter()
+        .for_each(|item| *counts.entry(item).or_insert(0) += 1);
+    b.unwrap_list()
+        .iter()
+        .for_each(|item| *counts.entry(item).or_insert(0) += -1);
```

Comment:
For array contains, Postgres seems to ignore duplicate elements, e.g.
```
parker.timmerman=# SELECT ARRAY[1,4,3] @> ARRAY[3, 3, 3, 1];
 ?column?
----------
 t
```

I think in this case the current implementation would return `false` because we'd end up with a count of `-2` for the element "3"?

Reply:
> In which world does that make sense?

I guess the Postgres world 🙃 I was surprised too, but I only checked because it felt like one of those things that might exist in a system built in the 90s haha